### ISSUE TITLE
Exploring Autonomous Agents and How you can make yours!

### ISSUE THREAD

**Issue by programmingninjas** [pydelhi/talks#260]
### Title

Exploring Autonomous Agents and How you can make yours!

### Describe your Talk

In this session, we will delve into the fascinating realm of advanced text generation, focusing on my innovative project "GPT 3.5 on Steroids." This project combines the power of GPT models with custom Python tools, enabling dynamic web scraping, language processing, and data retrieval. We will explore how this integration opens doors to unique and creative applications, setting it apart from traditional chatbots.

During the talk, I will showcase the project's capabilities, walk you through its architecture, and demonstrate real-world applications. By the end of this session, you will gain insights into the potential of advanced text generation and understand how to harness these technologies for your projects.

### Pre-requisites & reading material

A basic understanding of Python programming and Prompt Engineering  concepts will be beneficial for this talk.

### Time required for the talk

30 mins

### Link to slides/demos

https://github.com/programmingninjas/GPT-3.5-ON-STEROIDS

### About you

I'm a Full Stack Developer with bonus knowledge in domains like AI,ML,XR,Web3, Automation and Prompt Engineering.
2x Hackathon Winner 🏆 | 1x Hackathon Mentor 👨‍🏫 | SIH'22 National Finalist

- [LinkedIn](https://www.linkedin.com/in/ayankhan3105/)
- [Twitter](https://twitter.com/ayankhanstwt)

### Availability

Weekend

### Any comments

I would like you to go to my twitter and check recent tweets about the project videos of the same.

**Comment by Animesh-Ghosh** [comment]
Hey @programmingninjas would you be available on the 16th of December, 2023, to deliver your talk?

**Comment by programmingninjas** [comment]
I'm really sorry @Animesh-Ghosh but i will be occupied with my end sems from 12th december. I will let you know as soon as i get free again.

**Comment by pulsar17** [comment]
@programmingninjas will you be available to give the talk on 21st January, 2024?

**Comment by programmingninjas** [comment]
@pulsar17  Yes, I can

**Comment by pulsar17** [comment]
@programmingninjas please do confirm the new date in my comment above. (I typed the wrong date earlier, the meetups are supposed to happen on 3rd Saturday of every month)

**Comment by programmingninjas** [comment]
Yeah i will be available for 20 Jan. Let me know all the details about venue and timings.

**Comment by pulsar17** [comment]
@programmingninjas your talk is scheduled on 20th!
You can find the venue and timings in https://www.meetup.com/pydelhi/events/298483033/

**Comment by pulsar17** [comment]
@programmingninjas please either share your Telegram username or DM me on Telegram so that there is a way to contact you tomorrow. (My Telegram username is @pulsar17)

**Comment by pulsar17** [comment]
Thanks for giving a talk @programmingninjas ! Closing this issue now.
